### PR TITLE
Add digital ocean spaces link for S3 providers (and tidy up)

### PR DESCRIPTION
**Pull request by Meetcpatel**
This PR includes small information to use S3 comaptible storage services from Digital Ocean spaces

### CODE REVIEW

**Comment by jhohlfeld** [comment]
Hi @Meetcpatel,

may I ask how you managed to set your permissions from 'Private' to 'Public' on DigitalOcean? I was trying to set the ACL of newly uploaded assets automatically and found no way to do it out of the box, so I created #8264.

Is there any other way I am not aware of?

cheers
Jakob

File: docs/pages/docs/guides/images-and-files.md [hunk 1]
```diff
@@ -3,11 +3,11 @@ title: "Images and Files"
...
 
 ## How asset storage works in Keystone
 
-Keystone manages file and image assets through a `storage` object you define in Keystone’s [configuration file](/docs/config/config). Any number of stores can be set up within the `storage` object, and you can mix and match between `local` and `s3` (by [Amazon](https://aws.amazon.com/s3/)) depending on your use case.
+Keystone manages file and image assets through a `storage` object you define in Keystone’s [configuration file](/docs/config/config). Any number of stores can be set up within the `storage` object, and you can mix and match between `local` and `s3` (by [Amazon](https://aws.amazon.com/s3/)) and `S3-Compatible Cloud Object Storage` (by [DigitalOcean Spaces](https://www.digitalocean.com/products/spaces)) depending on your use case.
```
Comment (dcousens) [inline docs/pages/docs/guides/images-and-files.md:10]:
```suggestion
Keystone manages file and image assets through a `storage` object you define in Keystone’s [configuration file](/docs/config/config). Any number of stores can be set up within the `storage` object, and you can mix and match between `local` and `s3` compatible providers (akin to [Amazon S3](https://aws.amazon.com/s3/, or [DigitalOcean Spaces](https://www.digitalocean.com/products/spaces)).
```

We have no preference for a provider, only that it is compatible.  Maybe this is an improvement?

File: docs/pages/docs/guides/images-and-files.md [hunk 2]
```diff
@@ -66,6 +94,8 @@ storage: {
     accessKeyId,
     // The S3 Secret pulled from the S3_SECRET_ACCESS_KEY environment variable
     secretAccessKey,
+    // the S3 endpoint pulled from S3_ENDPOINT environment variable
```
Comment (dcousens) [inline docs/pages/docs/guides/images-and-files.md:97]:
```suggestion
    // the S3 endpoint pulled from S3_ENDPOINT environment variable (if defined)
```

File: docs/pages/docs/guides/images-and-files.md (hunk 1 above)
Comment (dcousens) [reply docs/pages/docs/guides/images-and-files.md:10]: I'm weary of adding many S3 providers to this section, but don't want to bias either.  Maybe we can use generic language after _one_ section that lists a number of _different_ providers.

**Comment by Meetcpatel** [comment]
> Hi @Meetcpatel,
> 
> may I ask how you managed to set your permissions from 'Private' to 'Public' on DigitalOcean? I was trying to set the ACL of newly uploaded assets automatically and found no way to do it out of the box, so I created #8264.
> 
> Is there any other way I am not aware of?
> 
> cheers Jakob

Sure, 
Yes, it stores as private asset on DigitalOcean, but I think for our use case we don't  need to make it public  because the asset link that we have after keystone uploads the asset is fine to access the same asset.

But, if you have other use case and it requires to make all asset public, I think that your idea would be helpful to make it public.

asset link from Digital Ocean: 
`https://bucketname.region.digitaloceanspaces.com/1f7fcab9-39f3-47c1-bbdc-0b5938bbc495.jpg` (we can't access, it gives access denied)

asset link keystone received after upload: 
`https://bucketname.region.digitaloceanspaces.com/1f7fcab9-39f3-47c1-bbdc-0b5938bbc495.jpg?X-Amz-Algorithm....` (we can access)

File: docs/pages/docs/guides/images-and-files.md (hunk 1 above)
Comment (Meetcpatel) [reply docs/pages/docs/guides/images-and-files.md]: Yes, Thank You!

**Review by dcousens — APPROVED** [review]

**Pull request merged**